docs.md

```diff
@@ -38,6 +38,13 @@ Base URL: https://nazareth-open-tourism-platform.herokuapp.com/
 ### Get all events
 `GET /events`
 
+**Query parameters**
+
+Name | Type | Description
+--- | --- | ---
+date_from | date string (e.g. 2017-08-20) | All returned events should have a startTime later than or equal to 00:00 on this date, if supplied.
```

Comment:
The format of this string follows a spec: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Either link this in or explain the options in more detail..